docs/ReleaseNotes.md

```diff
@@ -24,7 +24,7 @@ The Guava version has been updated to version 27. Users of the [shaded variants]
 * **Performance** Improvement 3 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Performance** Improvement 4 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Performance** Improvement 5 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
-* **Feature** Feature 1 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
+* **Feature** Added a counter counting indexes need to be rebuilt and changed related loggings to DEBUG [(Issue #604)](https://github.com/FoundationDB/fdb-record-layer/issues/604)
```

Comment:
```suggestion
* **Feature** Added a counter counting indexes that need to be rebuilt and changed related logging to `DEBUG` [(Issue #604)](https://github.com/FoundationDB/fdb-record-layer/issues/604)
```

Reply:
Done.

Reply:
Sorry for the nitpicking, but it looks like the "that" was added, but "loggings" -> "logging" and tick-marks around "DEBUG" were omitted.

Reply:
Thanks for pointing out! Fixed.